realm/realm-library/src/main/cpp/io_realm_internal_objectstore_OsMutableSubscriptionSet.cpp

```diff
@@ -43,9 +43,16 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_objectstore_OsMutableSubscription
 {
     try {
         auto subscriptions = reinterpret_cast<sync::MutableSubscriptionSet*>(j_subscription_set_ptr);
-        JStringAccessor name(env, j_name);
         auto query = reinterpret_cast<Query*>(j_query);
-        std::pair<sync::SubscriptionSet::iterator, bool> result = subscriptions->insert_or_assign(name, *query);
+        std::pair<sync::SubscriptionSet::iterator, bool> result;
+
+        if(j_name == nullptr) {
```

Comment:
Since we already mapped j_name here, I would probably use `name.is_null()` instead to keep usage of j_name to the top of the method.

Reply:
Oh I didn't know about that method, we can use a ternary operator too.